layouts/shortcodes/button/about-atsign.html

```diff
@@ -0,0 +1,6 @@
+<a href="#" class="btn btn-primary btn-lg mr-3 mb-4" role="button" aria-pressed="true">
```

Comment:
The href attribute is missing a way to input the URL as a parameter to the shortcode. See [here](https://gohugo.io/functions/get/) for more info on how you can do that.

Reply:
Should I have the href passed as a parameter or would it be suitable to directly enter https://atsign.com in place of "#"?